en/lessons/ecto/basics.md

```diff
@@ -57,18 +57,18 @@ All communication to the database will be done using this repository.
 Set up a repository by running:
 
 ```shell
-$ mix ecto.gen.repo -r Example.Repo
+$ mix ecto.gen.repo -r Friends.Repo
 ```
 
 This will generate the configuration required in `config/config.exs` to connect to a database including the adapter to use.
 This is the configuration file for our `Example` application
 
 ```elixir
-config :friends, Example.Repo,
+config :friends, Friends.Repo,
   adapter: Ecto.Adapters.Postgres,
   database: "friends_repo",
   username: "postgres",
-  password: "",
+  password: "postgres",
```

Comment:
Maybe here we should specify here that postgres credentials may differ?

Reply:
@tajchumber this is the default password on most installations. but happy to remove it. 👍